config/jobs/kubernetes-sigs/sig-windows/release-1.17-windows.yaml

```diff
@@ -1,227 +0,0 @@
-presubmits:
```

Comment:
Is 1.17 deprecated now?

Reply:
https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md#non-active-branch-history

Reply:
Didn't see the comment above